deck.md

```diff
@@ -514,6 +525,16 @@ Stand up. Yes, it’s silly. Yes, it helps.
 
 ### **Activity** :sparkles:
 
+## What's the most useful<br>you learned today?
```

Comment:
Tiny typo?

 ## What's the most useful<br>you learned today? 
👇 
 ## What's the most useful<br>thing you learned today?

Reply:
Well spotted, thank you!